### ISSUE TITLE
PTN000552695 check probably should not be "regulation of..."

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#4792]
* **PTHR ID & PTN node:**

GO:0045944 | positive regulation of transcription by RNA polymerase II | IBA with S000003502 , PTN000552695 | Gaudet P et al. (2011)

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPAC31G5.19
I'm not 100 % here, but  I don't think that chaperones should be annotated to 
"regulation of transcription" terms   should they?
They should be annotated directly to the relevent chromatin remodelling terms
@pgaudet 
@colinlog 
is that correct?"

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by ValWood** [comment]
If so, would also apply to


GO:0006357 | regulation of transcription by RNA polymerase II | IBA with PTN001326511 , S000000966
https://www.pombase.org/gene/SPAC1783.05

**Comment by colinlog** [comment]
YTA7 is the Sc yeast homolog of Sp abo1 and Hs ATAD2 (interpro: IPR045199). The Bromodomain is typical of chromatin remodelling proteins.

Hinnebush 2016 (PMID: 26602697, nicely balanced abstract) implicated YTA7 in GCN4 dependent transcription induction at ARG gene promoters, in the same pathway as SWI/SNF(snf2) and SAGA(gcn5) in Sc. That says: transcription regulation in the same pathway as dbTFs. 

Protein complex biochemistry (2007, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2171912/) YTA7 is stoichiometrically binding histone octamers and is somteimes physically together with the SAS3 HAT as well as Dpb4-containing Isw2 complex (ATP-dependent SNF2-superfamily nucleosome slider known to act at gene promoters).

YTA7 regulates (negatively and positively) regulates yeast histone gene promoter activity as a function of the cell cycle (Brenda Andrews, PMID: 19683497, PMID: 22156209).

The human homolog ATAD2 has also been imlicated in gene promoter activity driven by the estrogen and by the androgen receptors and a similar study exists for the Drosophila homolog - for the ecdysterone receptor

It can promote both repression and induction of RNA polymerae II gene promoter transcription activity. It is fine with the BP  "transcription initiation-coupled chromatin remodeling GO:0045815". 

It is also fine with BP GO:0060260 regulation of transcription initiation by RNA polymerase II [Any process that modulates the rate, frequency or extent of a process involved in starting transcription from an RNA polymerase II promoter.] but that does not capture the chromatin remodelling aspect expressed by GO:0045815, which should be a child of GO:0060260, in fact.


Furthermore, this protein family is fine with  CC GO:1990453 nucleosome disassembly/reassembly complex as these AAA+ ATPases form (heteromeric) hexameric rings and this one is a histone chaperone (MF:  GO:0140713 histone chaperone activity)

**Comment by ValWood** [comment]
OK, I had annotated to 

GO:0045815 | transcription initiation-coupled chromatin remodeling
which is at odds with the 
GO:0060260 regulation of transcription initiation by RNA polymerase II
(unless these roles are in different contexts, in which case it is OK).

bear in mind that for GO if something is part_of a pathway, we don't use the "regulation of" annotation. )I.e we don't use "regulation of"  terms just to annotate an upstream effect. However, I still get confused where and when to use regulation terms.

Perhaps the other recent changes to the positioning of 
GO:0045815 | transcription initiation-coupled chromatin remodeling have addressed this. I will wait and see how it pans out in the annotation.  

In a couple of weeks  I will go through all of my remaining chromatin questions (and make that chromatin heat -map  using the tool you showed me in Padua). Then I hope to join the Friday call with any outstanding questions. I think most of my queries are now resolved!
